guides/common/modules/proc_exporting-report-templates-by-using-web-ui.adoc

```diff
@@ -0,0 +1,14 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="exporting-report-templates-by-using-web-ui"]
+= Exporting report templates by using {ProjectWebUI}
+
+[role="_abstract"]
+You can export report templates that you create in {Project} from {ProjectWebUI}.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Monitor* > *Reports* > *Report Templates*.
+. Locate the template that you want to export, and from the list in the *Actions* column, select *Export*.
+. Repeat this action for every report template that you want to download.
++
+An `.erb` file that contains the template downloads.
```

Comment:
potentially out of scope: This sentence feels off to me. Your call.

Reply:
Yes, it feels very off. Let's just drop it.